packages/design-system/src/components/Pagination/Page.tsx

```diff
@@ -0,0 +1,50 @@
+import Button from '../Button/Button';
+import React from 'react';
+
+export interface PageProps {
+  /**
+   * Defines the page number.
+   */
+  index: number;
+  /**
+   * Renders current page if true, other links if false.
+   */
+  isActive: boolean;
+  /**
+   * A callback function used to handle state changes.
+   */
+  onPageChange?: (evt: React.MouseEvent) => void;
+  /**
+   * Defines application-specific routing in url for links.
+   */
+  renderPageHref: string;
```

Comment:
I just noticed that this is a string. Since it's not a function, could this be named `href` instead?

Reply:
Updated!